File: test/benchmarks/lua/factors.lua
start = os.clock()

N = 50000000
factors = {}

for i = 1, N do
  if N % i == 0 then
    factors[#factors] = i
  end
end

seconds = os.clock() - start
print('elapsed: ' .. seconds .. 's | ' .. factors[#factors])

File: test/benchmarks/python/factors.py
from time import process_time as clock

start = clock()

N = 50000000
factors = []

for i in range(1, N + 1):
    if N % i == 0:
        factors.append(i)

seconds = clock() - start
print("elapsed: " + str(seconds) + "s | " + str(factors[-1]))
